docs/pages/programming/jdbc_api.md

```diff
@@ -65,6 +83,20 @@ An uber JAR is a JAR file which contains all its dependencies. To use the JAR wi
 export CLASSPATH=<path_to_the_uber_jar>:$CLASSPATH
 ```
 
+### With Gradle
```

Comment:
You might want to consider rewording [this](https://github.com/bloomberg/comdb2/pull/429/files#diff-81f465a33478484ae1aa5754496fea74R63). I suggest you change it to

> There are 2 approaches to set up cdb2jdbc: using build tools or setting the classpath.

and make Maven and Gradle under the section of build tools.